### PR TITLE
SDK: add support for multiple files

### PR DESCRIPTION
**Pull request by ggdaltoso**
As we can now have the possibility to open multiple files, and also multiple panes, in StackBlitz editor, I'm opening this PR to adjust the SDK and reflect those features.

Here are some links to demonstrate this new feature:
 - [Multiple files](https://stackblitz.com/edit/react-ts-s3lt6r?file=index.tsx,style.css,Button.tsx)
 - [Multiple files + Splitting panes](https://stackblitz.com/edit/react-ts-s3lt6r?file=index.tsx,style.css&file=Button.tsx)

### CODE REVIEW

File: sdk/src/helpers.ts [hunk 1]
```diff
@@ -27,6 +27,12 @@ export function buildProjectQuery(options?: EmbedOptions) {
       params.push(`file=${options.openFile}`);
     }
 
+    if (Array.isArray(options.openFile)) {
+      options.openFile.forEach((file) => {
+        params.push(`file=${file}`);
```
Comment (fvsch) [inline sdk/src/helpers.ts:32]:
```suggestion
        if (typeof file === 'string') {
          params.push(`file=${file}`);
        }
```

**Review by fvsch — COMMENTED** [review]
Looking pretty good.

Before releasing a new SDK version, we probably need to do some extra testing (and/or start writing a test suite :P). I might have some bandwidth for that early next week.

I’m also wondering if we should update `docs/readme.md`, or if we should remove the whole SDK section there and point to https://developer.stackblitz.com/docs/platform/javascript-sdk/ instead. Any thoughts @sulco @sylwiavargas?

File: sdk/src/helpers.ts (hunk 1 above)
Comment (sulco) [reply sdk/src/helpers.ts:32]: I think it's a good point, @fvsch. I would retire this section in favor of the developer.stackblitz site – we already treat it as the primary source of documentation, and maintaining both doesn't seem to have much upside.

**Review by fvsch — CHANGES_REQUESTED** [review]
Can you revert the changes to `docs/readme.md` @ggdaltoso?

**Comment by fvsch** [comment]
I'm handling removing the old docs in #1759.

**Comment by HeyGarrison** [comment]
@fvsch can we get this one in and then work on test suites? I'd hate to hold this one up indefinitely.

**Comment by fvsch** [comment]
Quick note: the multiple file param feature on stackblitz.com was reverted, so we should wait for it to land again before adding it to our SDK.

**Review by fvsch — APPROVED** [review]
Issue fixed on stackblitz.com, let's go.

**Pull request merged**